sdk/core/azure-core-amqp/test/ut/message_source_target.cpp

```diff
@@ -47,7 +47,16 @@ namespace {
 std::string timeToString(std::chrono::system_clock::time_point t)
 {
   std::time_t time = std::chrono::system_clock::to_time_t(t);
+#ifdef _MSC_VER
```

Comment:
this is also tests file. Should all the tests in the `ut` directory have `_tests` file suffix?

Reply:
Adding _tests to a file in a test directory is redundant.

Reply:
that is not the current practice, even some files you have do have `_tests` suffix.